llvm/test/tools/llvm-objcopy/ELF/change-section-lma-multiple-load.test

```diff
@@ -0,0 +1,52 @@
+# RUN: yaml2obj %s -o %t
+# RUN: llvm-objcopy --change-section-lma *+0x20 %t %t2
+# RUN: llvm-readelf -l %t2 | FileCheck %s --check-prefix=CHECK-PLUS
+# RUN: llvm-objcopy --change-section-lma *-0x10 %t2 %t3
+# RUN: llvm-readelf -l %t3 | FileCheck %s --check-prefix=CHECK-MINUS
+# RUN: not llvm-objcopy --change-section-lma .text3=0x5000 %t  2>&1 | FileCheck %s --check-prefix=ERR
+
+!ELF
+FileHeader:
+  Class:           ELFCLASS64
+  Data:            ELFDATA2LSB
+  Type:            ET_EXEC
+  Machine:         EM_AARCH64
+Sections:
```

Comment:
If I'm not mistaken, I think the sections are actually irrelevant to the test case and code behaviour (you can explicitly set the p_filesz of the segment in the YAML, without needing sections to drive it). If so, let's omit them.

Reply:
When I removed the sections the executable generated by yaml2obj was not usable by llvm-objcopy, although it looked quite similar. 

` error: program header with offset 0x1000 and file size 0x2000 goes past the end of the file`

Reply:
Okay, fair enough. I've actually changed my mind about not having any sections anyway, as I feel like it would be good to verify that the section sh_addr value isn't changed, as part of "documenting what the feature does through tests".

Reply:
Hopefully the latest version of the test shows this.